src/drivers/gps/gps.cpp

```diff
@@ -909,6 +909,15 @@ GPS::run()
 
 		gpsConfig.interface_protocols = static_cast<GPSHelper::InterfaceProtocolsMask>(gps_ubx_cfg_intf);
 
+		int32_t gps_cfg_wipe = 0;
+		handle = param_find("GPS_CFG_WIPE");
+
+		if (handle != PARAM_INVALID) {
+			param_get(handle, &gps_cfg_wipe);
+		}
+
+		gpsConfig.cfg_wipe = static_cast<bool>(gps_cfg_wipe);
```

Comment:
I think this might be be better as a toggle. The user would set this parameter to 1 and reboot. Upon reboot the driver would erase the flash and then set this parameter back to 0.